docs/development/front-end.rst

```diff
@@ -0,0 +1,158 @@
+Front End Development
+=====================
+
+Background
+----------
+
+.. note::
+
+    Consider this the canonical resource for contributing Javascript and CSS. We
+    are currently in the process of modernizing our front end development
+    procedures. You will see a lot of different styles around the code base for
+    front end JavaScript and CSS.
+
+Our modern front end development stack includes the following tools:
```

Comment:
Using the word modern here and in the note is a bit confusing. Are we planning on changing or is this what we are changing to?

Reply:
This page was only moved by this PR, i'm not focusing on this page ATM and it will change with design work.